libs/ui/tabs/brain/src/lib/brn-tabs-list.directive.ts

```diff
@@ -25,14 +25,13 @@ export class BrnTabsListDirective implements AfterContentInit {
 
 	private _keyManager?: FocusKeyManager<BrnTabsTriggerDirective>;
 
-	@ContentChildren(BrnTabsTriggerDirective, { descendants: true })
-	public triggers?: QueryList<BrnTabsTriggerDirective>;
+	public triggers = contentChildren(BrnTabsTriggerDirective, { descendants: true });
 
 	public ngAfterContentInit() {
 		if (!this.triggers) {
 			return;
 		}
```

Comment:
```suggestion
```

@marcjulian I guess what I am saying is that to me it looks like since this is a signal it will never be undefined and this check doesn't make any sense and we can remove it

Reply:
@goetzrobin thanks, I completely missed that! I will remove it and will double check #461 as well.